src/v2/view-builder/views/signin/SignInWithDeviceOption.js

```diff
@@ -4,17 +4,37 @@ import hbs from 'handlebars-inline-precompile';
 export default View.extend({
   className: 'sign-in-with-device-option',
   template: hbs`
-    <div class="okta-verify-container"></div>
+    <div class="okta-verify-container">
+    {{#if signInWithDeviceIsRequired}}
+      <div class="signin-with-ov-description">
+        {{i18n code="oktaVerify.description" bundle="login"}}
+      </div>
+    {{/if}}
+    </div>
+    {{#if signInWithDeviceIsNotRequired}}
     <div class="separation-line"><span>OR</span></div>
+    {{/if}}
   `,
   initialize () {
     const appState = this.options.appState;
     this.add(createButton({
       className: 'button',
       title: loc('oktaVerify.button', 'login'),
       click () {
-        appState.trigger('invokeAction', 'launch-authenticator');
+        if (this.options.isRequired) {
+          this.options.form.trigger('save', this.options.form.model);
```

Comment:
Could be

```
    this.options.appState.trigger('saveForm', model);
```

and avoid to pass `form`

Reply:
I can make the change. But form is available in options, I don't need to pass in for this view specifically.

Reply:
looks like. it's form the view. 
```
{ isRequired: .., form: this }
``